### DISCUSSION TITLE
GitHub achievements

### DISCUSSION THREAD

**Discussion by kimsh153** [community/community#18774] [Other Feature Feedback, Questions, & Ideas]
how to get
![](https://cdn.discordapp.com/attachments/820873397113585674/987192667609894972/2022-06-17_12.11.10.png)
this badge?

**Comment by baekteun** [comment]
Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)

**Comment by melvincwng** [comment]
@kimsh153, the Arctic Code Vault Contributor badge cannot be obtained as of today. It was available for developers to obtain in 2020 if they contributed code in a valid repository of the [2020 GitHub Archive Program](https://archiveprogram.github.com/).

Hope this clarifies.

**Comment by JonasJs** [comment]
Arctic Code Vault Contributor badge cannot be obtained.
It was for whoever Contributed code to repositories in 2020
[2020 GitHub Archive Program](https://archiveprogram.github.com/)

> To understand in more detail about other achievements, access this repository.
> https://github.com/Schweinepriester/github-profile-achievements

**Comment by ApidriuC** [comment]
No longer obtainable, it was a badge for contributors to certain repositories in 2020
